CitiesSkylinesMods/TMPE
# record vehicle restrictions

Added record for vehicle restriction.

I added 2 methods to the vehicle restriction manager to be able to clear vehicle restrictions for a lane and to be able to understand if vehicle restriction data is empty in which case I don't store anything.

### Test:
use one of the following methods:
A) You can also test this by using the clear/undo button from the road selection panel:
 - set up vehicle restrictions
 - use the clear button to clear traffic rules
 - make sure vehicle restrictions  are cleared
 - Press the clear button again to UNDO traffic rules.
 - Expected result: Vehicle restrictions are back up

B) put vehicle restrictions in the asset interchange editor and place them in-game

C) use move it mod to copy vehicle restriction. this [intermediate version of moveIt](https://github.com/Quboid/CS-MoveIt/files/4805542/MoveIt.zip) is suitable for this review:


![Screenshot (1137)](https://user-images.githubusercontent.com/26344691/85941096-6ce58400-b929-11ea-8577-2410e2d04276.png)
